_includes/sections/browser-tweaks.html

```diff
@@ -125,6 +125,9 @@ <h3>Getting started:</h3>
   </ul>
   </dd>
 
+  <dt>extensions.pocket.enabled = false</dt>
+  <dd>Disables pocket completely.</dd>
```

Comment:
```suggestion
  <dd>Disables Pocket completely.</dd>
```
And maybe add why it should be disabled?